ui/js/constants/settings.js

```diff
@@ -1,5 +1,6 @@
+/*hardcoded names still exist for these in reducers/settings.js - only discovered when debugging*/
 export const CREDIT_INTRO_ACKNOWLEDGED = "credit_intro_acknowledged";
-export const FIRST_RUN_ACKNOWLEDGED = "welcome_acknowledged";
+export const NEW_USER_ACKNOWLEDGED = "welcome_acknowledged";
```

Comment:
Shouldn't the value for this be changed to `new_user_acknowledged` or is there a reason for sticking with `welcome_acknowledged`?

Reply:
Yes, the value locally stored in the app by the constant value. Thus, changing the value would invalidate existing settings. I added a comment to settings.js about this, since it applies to many settings.